includes/fbbackground.php

```diff
@@ -148,6 +150,8 @@ protected function task( $item ) {
 		 * performed, or, call parent::complete().
 		 */
 		protected function complete() {
+			WC_Tracks::record_event( 'product_sync_background_sync_completed' );
```

Comment:
Whoa what is this class? Is this another product sync background process? I thought correct background process was `\SkyVerge\WooCommerce\Facebook\Products\Sync\Background`. @budzanowski?

Reply:
Uh oh!? I wondered about this too .. in any case, this event is just an example, if we move forward with tracking can remove or rename this event.